wp-content/themes/access/single-location.php

```diff
@@ -25,7 +25,7 @@
 enqueue_inline('google-tag-manager');
 
 // Main
-enqueue_script('main');
+enqueue_script('single-location');
```

Comment:
This file needs to be processed by Gulp/Webpack and compiled into the `assets/js` directory before it will work. You can run `npm run scripts` to process scripts. It will delete and compile all scripts in the distribution but you can selectively commit the **single-location.{{ hash }}.js** file that will appear to this branch after running the command.

Reply:
@devowhippit When I added the mobile `iframe` it seems to be blocking the iframe for Desktop. I'm guessing that is somehow firing only once.
```
    <section>
      <div data-js="google-static-map"
           data-google-static-map="https://www.google.com/maps/embed/v1/place?&center={{ post.address.lat}},{{ post.address.lng }}&q={{ post.address.address|url_encode }}&key={{ function('constant', 'GOOGLE_MAPS_EMBED') }}&zoom=15"
           class="block bg-blue-light border border-grey-light print:hidden screen-desktop:hidden"
           width="100%"
           height="300" 
           frameborder="0"></div>
    </section>
```

I thought that the Self Invoking function would be trigger for both elements but that doesn't seem to be the case. I'm currently trying to figure out how to make the self invoking function trigger both `div` and convert them to iframes.